packages/cli/commands/upload.js

```diff
@@ -214,5 +279,20 @@ exports.builder = yargs => {
     describe: i18n(`${i18nKey}.positionals.dest.describe`),
     type: 'string',
   });
+  yargs.option('options', {
+    describe: i18n(`${i18nKey}.positionals.options.describe`),
+    type: 'array',
+    default: [''],
+  });
+  yargs.option('saveOutput', {
+    describe: i18n(`${i18nKey}.positionals.options.saveOutput`),
+    type: 'boolean',
+    default: true,
+  });
+  yargs.option('processFields', {
+    describe: i18n(`${i18nKey}.positionals.options.processFields`),
```

Comment:
Looks like this string might be missing.

On the other hand, do we want to hide this flag? Unsure of rollout plan for this, but happy to chat over Slack

Reply:
Whoops, it should have been `positial.processFields.describe`. Fixed. And yeah, I think it's a good idea to hide these for now.